modules/get-started/pages/licensing/monitor-license-status.adoc

```diff
@@ -147,6 +147,40 @@ Redpanda Console displays warnings in the following scenarios:
 
 - *Redpanda Self-Managed enterprise features are in use without a valid license*: If the Redpanda cluster is using enterprise features without a valid license, Redpanda Console displays a warning specifying the features in use without proper licensing.
 
+== Check the license status on the Redpanda resource in Kubernetes
+
+If you have deployed a Redpanda cluster in Kubernetes using the Redpanda Operator, you can monitor the Enterprise Edition license status from the Redpanda custom resource. The operator integrates license monitoring into the resource's `status` field, allowing you to query the license state, expiration, and active enterprise features with `kubectl`.
+
+. Identify the Redpanda resource:
++
+[,bash]
+----
+kubectl get redpanda
+----
++
+Example output:
++
+[.no-copy]
+----
+NAME                  AGE
+example-redpanda      12m
+----
+
+. Check the license status:
++
+[,bash]
+----
+kubectl get redpanda <resource-name> -o jsonpath='{.status.licenseStatus}'
+----
++
+Replace `<resource-name>` with the name of your Redpanda resource, such as `example-redpanda`.
++
+The `status.licenseStatus` field includes:
++
+- `valid`: Indicates whether the license is valid (true) or not (false).
+- `expiryDate`: The date the license will expire (format: YYYY-MM-DD).
+- `features`: A list of enterprise features currently in use.
```

Comment:
Just FYI, other than valid, we don't show this information in Kubernetes, and valid is just a status condition, there's no `licenseStatus` part of the CRD, we only publish status information under the `ClusterLicenseValid` condition in `status.conditions`.

We could add additional details if we want to, but that's not how this currently works. @david-yu thoughts about extending the status to incorporate some of these or is just publishing how to check these via execing and running `rpk cluster license info` enough since we publish the non-granular license status in the CRD?

Reply:
@andrewstucki It would be nice to include those fields as well. I don't know if its possible to do another release of the Operator by Tuesday that includes those fields?

Reply:
Also is this status going to be updated a lot or will only happen maybe like once an hour or so?

Reply:
Ok lets see how it goes through this week. Perhaps Rafal can help review if a PR is available by tomorrow?

Reply:
Created a follow-up issue for docs: https://redpandadata.atlassian.net/browse/DOC-868